src/components/ColorPicker.vue

```diff
@@ -55,7 +55,7 @@
           :value="value"
           :placeholder="placeholder"
           @input="updateFromInput"
-          v-on="listeners"
+          v-on="$listeners"
```

Comment:
hey @joshua-larks, which one is correct? wasn't sure when resolving conflict

Reply:
For my changes I am using it without `$`. I made a computed property named `listeners` that takes `$listeners` and removes the `input` event listener. 

listener listener listener listener listener